app/models/plate.rb

```diff
@@ -455,6 +455,8 @@ def update_qc_values_with_parser(parser)
     true
   end
 
+  # Finds the product line (= team) of the requests coming out of this plate's 'stock plate'.
+  # Written at a time when requests weren't recorded on the aliquot, so could be re-written in a less convoluted way.
```

Comment:
Again I think is historical description which is interesting but seems difficult to maintain over new changes in future.